iidm/iidm-api/src/main/java/com/powsybl/iidm/network/AcDcConverter.java

```diff
@@ -229,4 +241,19 @@ enum ControlMode {
      * Get the target DC voltage (kV DC)
      */
     double getTargetVdc();
+
+    /**
+     * Create a Droop curve;
+     */
+    DroopCurveAdder newDroopCurve();
+
+    /**
+     * Set the Droop curve;
+     */
+    void setDroopCurve(DroopCurve droopCurve);
```

Comment:
This method should not be part of the API, just keep it on the implementation side (and no problem in the impl side if you need to cast)